cli/download_source.go

```diff
@@ -18,6 +18,7 @@ import (
 
 // manifest for files copied from terragrunt module folder (i.e., the folder that contains the current terragrunt.hcl)
 const MODULE_MANIFEST_NAME = ".terragrunt-module-manifest"
+const MODULE_NEED_INIT = ".terragrunt-init-required"
```

Comment:
Style NIT: We've been switching to canonical golang consts of using `camelCase` instead of `ALL_CAPS`. Can you switch this to camelCase?

Reply:
Also, there should be a new line between this const and the `MODULE_MANIFEST_NAME` const. Otherwise, as is, the comment looks like it applies to both consts which isn't correct.